### PR TITLE
Rearrange parts in README.md to and remove call for maintainers

### PR DESCRIPTION
**Pull request by mathbunnyru**
## Checklist (especially for first-time contributors)

- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] I will try not to use force-push to make the review process easier for reviewers
- [ ] I have updated the documentation for significant changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,44 +69,13 @@ So, new notebooks will be saved there, unless you change the directory in the fi
 To change the default directory, you will need to specify `ServerApp.root_dir` by adding this line to previous command: `start-notebook.py --ServerApp.root_dir=/home/jovyan/work`.
 ```
 
-## Contributing
-
-Please see the [Contributor Guide on ReadTheDocs](https://jupyter-docker-stacks.readthedocs.io/en/latest/)
-for information about how to contribute recipes, features, tests, and community maintained stacks.
-
-## Maintainer Help Wanted
```
Comment (mathbunnyru) [inline README.md:77]: This section doesn't make a lot of difference.

I want to move user-related sections of the readme up, so people first see how to _use_ our images, and then how to _contribute_ and what alternatives exist.

**Review by yuvipanda — APPROVED** [review]
If you feel the 'call for maintainers' can be removed, that feels like a huge milestone in the health of the project!

**Comment by mathbunnyru** [comment]
> If you feel the 'call for maintainers' can be removed, that feels like a huge milestone in the health of the project!

I do believe the project is in a much healthier state right now than it was a few years. Lots of fix have been fixed, not so many new issues were added.
At the same time, I would love to see new contributors here - it would be nice to have more people working on this project.

**Comment by yuvipanda** [comment]
@mathbunnyru fwiw I've started 'watching' this project a few weeks ago as well so I can contribute not just in terms of new features but in other ways too

**Comment by mathbunnyru** [comment]
> @mathbunnyru fwiw I've started 'watching' this project a few weeks ago as well so I can contribute not just in terms of new features but in other ways too

It would definitely be great 👍 🔥

**Pull request merged**